app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-enhanced-ecommerce-tracker.js

```diff
@@ -40,7 +40,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         })
       }
     }
-    
+
```

Comment:
Could these linting fixes be merged with the commit that added these files?

Reply:
Done.